SECURITY.md

```diff
@@ -0,0 +1,28 @@
+# RIOT Security Policy
+
+All security bugs reported will be silently fixed in `master` and backported
+to the previous release.
+
+## Reporting a Vulnerability
+
+If a security issue is discovered, please report it to security@riot-os.org.
+A response will be provided within one week.
+The issue will be tracked in the [security mailing list](security@riot-os.org).
+The original reporter will be included in the discussion of the issue.
+
+## Notification of a Vulnerability
+
+After a fix is provided the security issue will be privately disclosed to the
+original reporter, RIOT security maintainers, and "Trusted RIOT Users".
+A public announcement of the security fix will be made two weeks after the
+point release, though this may vary depending on the severity and ability of
+trusted RIOT users to provide the fix.
+
+## Trusted RIOT Users
+
+To join the "Trusted RIOT Users" notifications list please send information
+on the RIOT based service or product to the
+[security mailing list](security@riot-os.org).
```

Comment:
Why being so vague? IIRC we discussed offline that this "Trusted RIOT Users" notification list will be a dedicated category at https://forum.riot-os.org. IMHO we can (and should) be very open about this. As such, you should also mention, that we need their forum username with the information they send.